site/src/component/ChangelogModal/ChangelogModal.scss

```diff
@@ -1,21 +1,10 @@
 @use '../../globals';
 
 .changelog-modal {
-  .modal-dialog {
+  .MuiDialog-paper {
     max-width: 500px;
   }
```

Comment:
i think you're able to give the Paper a className via `slotProps.paper.className`, and that can be used for the outer css selector here (`.changelog-modal`)